helm/oncall/values.yaml

```diff
@@ -269,6 +271,9 @@ grafana:
   plugins:
     - grafana-oncall-app
 
+externalGrafana:
+  url:
```

Comment:
this is the main change in this file, the rest is `yaml` IDE autoformatter changes